build-env/install-prereqs.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+ZSDK_VERSION=0.11.2
+
+# install prerequisite packages:
+dpkg --add-architecture i386
+
+apt-get update && apt-get upgrade -y
+
+DEBIAN_FRONTEND=noniteractive apt-get install -y --no-install-recommends \
+    ca-certificates \
+    ccache \
+    cmake \
```

Comment:
why install cmake here when you later on get it directly from kitware?

Reply:
IIRC, I was scaling back the already existing tooling which may have had it?  

In any event, I wasn't aware what's installed from kitware.